app/controllers/claims/pages_controller.rb

```diff
@@ -1,2 +1,5 @@
 class Claims::PagesController < ApplicationController
+  def index
+    @user = User.first
```

Comment:
I've just realised. Is this the correct page?

The organisation details should be for the `/schools/:school_id` page, right?